typegate/src/graphql/graphql.ts

```diff
@@ -98,6 +103,7 @@ export function parseGraphQLTypeGraph(typegraph: TypeGraphDS) {
       ...rootNode,
       title: "Query",
       properties: queryProperties,
+      config: { ...(rootNode.config ?? {}), __namespace: true },
```

Comment:
It was missing, though all the other namespaces have it. See "Mutation" just below.
It did not cause any problem since it is the root namespace, but I think we might need it in the future.